src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@patternfly/react-core';
+import { TrashIcon, PencilAltIcon, FileIcon } from '@patternfly/react-icons';
+
+import styles from './ActionButton.module.css';
+
+const actionIcons = {
+  create: <FileIcon />,
+  edit: <PencilAltIcon />,
+  delete: <TrashIcon />,
+};
+
+interface IActionButton {
```

Comment:
In the end we decided to change naming guidelines a little bit, please use `IActionButtonProps` instead (the same is valid for all other places)